mark.go

```diff
@@ -0,0 +1,46 @@
+package qt
+
+import (
+	"runtime"
+	"testing"
+)
+
+// A Marker records a name at a given Go source file position.
+type Marker struct {
+	_    struct{} // Prevent
```

Comment:
Prevent what? Building this without specifying the field names? If so, please expand the comment, and explain the reason?